ui/app/pages/create-account/connect-hardware/account-list.js

```diff
@@ -104,8 +121,8 @@ class AccountList extends Component {
                   <span className="hw-account-list__item__index">
                     {account.index + 1}
                   </span>
-                  {`${account.address.slice(0, 4)}...${account.address.slice(
-                    -4,
+                  {`${account.address.slice(0, 5)}...${account.address.slice(
```

Comment:
I'm curious what motivated this change. The most recent Ledger Live designs I saw still used the first and last 4 characters of the address, rather than 5.

Reply:
Relic of the original PR, restored to 4.